plugins/inputs/consul/README.md

```diff
@@ -38,6 +38,17 @@ report those stats already using StatsD protocol if needed.
   # When tags are formatted like "key:value" with ":" as a delimiter then
   # they will be splitted and reported as proper key:value in Telegraf
   # tag_delimiter = ":"
+
+  ## Disable gathering tags from Consul on health checks
+  # This is very useful on large clusters with a lot of services and tags.
+  # This alleviates the situation of this telegraf running on multiple servers 
+  # for a holistic point of view, and there are thousands of checks with 10+ tags. 
+  # disable_tags = false
```

Comment:
Do you think you could update this to match the method I suggested here: https://github.com/influxdata/telegraf/issues/7387#issuecomment-623724745

You can set this up fairly easily by using `filters.NewIncludeExcludeFilter`.

Reply:
Sure!  So we're on on the same page, to replicate the behaviour I want, I'd need to set:

```
[[inputs.consul]]
  ... # config
  service_tag_include = []
  service_tag_exclude = ["*"]
```

The behaviour I'm expecting would be for this to pass along service_id, node, and check_id, where check_id can be excluded with `taxexclude = ["check_id"]` to replicate what I've tried here.

Is my understanding correct?

Reply:
Yes, that's right.  To clarify the behavior of the `service_tag_include` and `service_tag_exclude` options, they would match the entire ServiceTag string, before any splitting, but would only affect these tags.